front/src/applications/stdcm/hooks/useStdcmResults.ts

```diff
@@ -59,12 +59,12 @@ const useStdcmResults = (
     const getPathProperties = async (_infraId: number, path: PathfindingResultSuccess) => {
       const pathPropertiesParams: PostInfraByInfraIdPathPropertiesApiArg = {
         infraId: _infraId,
-        props: ['electrifications', 'geometry', 'operational_points'],
+        props: ['electrifications', 'geometry', 'operational_points', 'zones'],
         pathPropertiesInput: {
           track_section_ranges: path.track_section_ranges,
         },
       };
-      const { geometry, operational_points, electrifications } =
+      const { geometry, operational_points, electrifications, zones } =
```

Comment:
Check the comment in operationamStudies/types.ts

```suggestion
      const { geometry, operational_points, zones } =
```